src/index.js

```diff
@@ -1,5 +1,21 @@
 import './style.css';
-import { added } from './module/add.js';
+import postScores from './postScores.js';
+import getScores from './getScores.js';
+import showScores from './showScores.js';
+import showError from './showError.js';
 
-const add = document.querySelector('.add');
-add.addEventListener('click', added);
\ No newline at end of file
+const refresh = document.getElementById('refresh');
+const form = document.getElementById('form');
+const gameApi = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/GDUN7EKtNyCRpmru02cT/scores';
+
+refresh.addEventListener('click', () => {
+  getScores(gameApi).then((data) => showScores(data.result));
```

Comment:
- [ ] Good job working on this but please kindly ensure to use `async` and `await` for asynchronous function calls.